INC CREATEv1

```diff
@@ -0,0 +1,197 @@
+import unittest
+
+class Servicenow_IncCreate(unittest.TestCase):
+    def setUp(self):
+        self.driver = webdriver.Firefox()
+#        base_url =  "https://dev65721.service-now.com"
+        
+#        driver.implicitly_wait(10)
+
+#        
+
+#        driver.get(base_url)
+
+    def test_selenium(self):	
+        #
+        identifier = "AUTOMATED TESTING"
+        module = "INCIDENT"
+        action = "CREATE"
+        job_name = ""
+        today = str(date.today())
+        
+        driver = self.driver
+        
+        base_url =  "https://dev65721.service-now.com"
+        
+        driver.get(base_url)
+        
+        
+        driver.switch_to_frame("gsft_main")
+        username = driver.find_element_by_id("user_name")
+        username.send_keys("admin")
+
+        password = driver.find_element_by_id("user_password")
+        password.send_keys("Liverpool28")
+        password.send_keys(Keys.RETURN)
+
+        identifier_inc = ("AUTOMATED TESTING - INCIDENT - %s" %today)
+
+        # depreciated warning
+        driver.switch_to.default_content()
+        # driver.switch_to.frame
+
+        time.sleep (10)
+
+        # Left Nav, incident.do 
+        createnew = driver.find_element_by_id("filter")
+        createnew.send_keys("Create New")
+        time.sleep(1)
+        createnew.send_keys(Keys.RETURN)
+        time.sleep(1)
+
+        # switch to main frame to create the incident
+        driver.switch_to_frame("gsft_main")
+
+
+        try:
+            element = WebDriverWait(driver, 10).until(
+            EC.presence_of_element_located((By.ID, "incident.category"))
+                
+                )
+        except:
+            print("Element not found")
+                
+            #driver.quit()
+
+        category = driver.find_element_by_id("incident.category")
+        category.send_keys("Software")
+        time.sleep(1)
+
+
+        subcategory = driver.find_element_by_id("incident.subcategory")
+        subcategory.send_keys("Email")
+        time.sleep(1)
+
+        time.sleep(2)
+        callerbox = driver.find_element_by_id("sys_display.incident.caller_id")
+        callerbox.send_keys("Abraham Lincoln")
+        callerbox.send_keys(Keys.RETURN)
+        time.sleep(1)
+
+        contacttype = driver.find_element_by_id("incident.contact_type")
+        contacttype.send_keys("Email")
+        time.sleep(1)
+
+        impact = driver.find_element_by_id("incident.impact")
+        impact.send_keys("2")
+        time.sleep(1)
+
+        urgency = driver.find_element_by_id("incident.urgency")
+        urgency.send_keys("2")
+        time.sleep(1)
+
+        assign_grp = driver.find_element_by_id("sys_display.incident.assignment_group")
+        assign_grp.send_keys("Database")
+        assign_grp.send_keys(Keys.RETURN)
+
+        time.sleep(1)
+
+
+        short_desc = driver.find_element_by_id("incident.short_description")
+        short_desc.send_keys("%s %s %s %s" %(identifier, module, action, today))
```

Comment:
format is a lot easier to read:
 short_desc.send_keys("{} {} {} {}".format(identifier, module, action, today))